react-maskedinput/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for react-maskedinput 3.3
+// Project: https://github.com/insin/react-maskedinput
+// Definitions by: Karol Janyst <https://github.com/LKay>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
```

Comment:
These actually compile fine under `typescript@2.0` if you add a semicolon at the end of `export as namespace MaskedInput`.